app/views/layouts/_footer.html.erb

```diff
@@ -1,4 +1,3 @@
-
-    <div class="footer">
-      <p>Powered by <a href="http://www.agileventures.org">AgileVentures</a> volunteers &#9734; Open Source software</p>
-    </div>
+<div class="footer">
+  <p>Powered by <a href="http://www.agileventures.org">AgileVentures</a> volunteers &#9734; Open Source software</p>
+</div>
```

Comment:
I merely fixed the white space here, the footer contents are beyond the scope of this PR.  Shall we create a task?